src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/activities/AllocationResourceIntTest.kt

```diff
@@ -313,6 +324,28 @@ class AllocationResourceIntTest : IntegrationTestBase() {
           assertThat(it).contains("INVALID")
         }
     }
+
+    @Test
+    @Disabled("WIP")
```

Comment:
Turned all the tests off for now, just wanted to get them down first to keep future PRs smaller